include/xtl/xdynamic_bitset.hpp

```diff
@@ -152,37 +186,95 @@ namespace xtl
         block_type bit_mask(size_type pos) const noexcept;
         size_type count_extra_bits() const noexcept;
         void zero_unused_bits();
+    private:
 
-        storage_type m_buffer;
-        size_type m_size;
+        // Make views and buffers friends
+        template<typename BB>
+        friend class xdynamic_bitset_base;
     };
 
-    template <class B, class A>
-    xdynamic_bitset<B, A> operator~(const xdynamic_bitset<B, A>& lhs);
+    // NOTE this view ZEROS out remaining bits!
```

Comment:
Thus we cannot have `xdynamic_bitset_view` on const buffers. This is not blocking for now, but I think we should address this issue sooner or later.